nexus/src/external_api/http_entrypoints.rs

```diff
@@ -1643,7 +1643,7 @@ async fn instance_network_interfaces_get_interface(
     let handler = async {
         let opctx = OpContext::for_external_api(&rqctx).await?;
         let interface = nexus
-            .network_interface_fetch(
+            .instance_network_interface_fetch(
```

Comment:
I don't really care but the two renames in this file seem self-inconsistent?  Or: what's the naming convention implied by these renames?

Reply:
I'll just revert to the before-this-PR names.